### PR TITLE
Feature/evo 21

### PR DESCRIPTION
**Pull request by mrhodas107**
-The shipping information can now be viewed in OrderDetailsPage.
-Payment information can now be viewed in OrderDeatilsPage.
-The shipping information can now be viewed in OrderDetailsPage.

### CODE REVIEW

File: app/containers/OrderPage/OrderDetailPage/ShippingInformation/index.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+// remove, use babel for antD css
+import { Col, Card, Icon, Button, List } from 'antd';
+import 'antd/dist/antd.css';
+
+import * as React from 'react';
+
+import { shippingDetailData } from './constants';
+
+export default function OrderDetailHeader() {
```
Comment (jmiller-cvet) [inline app/containers/OrderPage/OrderDetailPage/ShippingInformation/index.tsx]: Is this the correct name for this function?  I think you meant "ShippingInformation".

File: app/containers/OrderPage/OrderDetailPage/index.tsx [hunk 2]
```diff
@@ -1,24 +1,37 @@
...
 
-/* eslint-disable react/prefer-stateless-function */
-export default function OrderDetailPage() {
+export default function OrderDetail() {
 
   return (
     <Layout>
-      <OrderDetail/>
+      <OrderDetailHeader/>
+      <Row gutter={16} align="top" style={{ marginTop: '20px' }}>
```
Comment (jmiller-cvet) [inline app/containers/OrderPage/OrderDetailPage/index.tsx]: Does this give us the desired behavior so this row becomes a column when the screen size is less than 1920?

File: app/containers/OrderPage/OrderDetailPage/ShippingInformation/index.tsx (hunk 1 above)
Comment (mrhodas107) [reply app/containers/OrderPage/OrderDetailPage/ShippingInformation/index.tsx]: Oh yes, you are correct.  The app will still work just fine.  All those 'information' containers are named incorrectly now that I realize.  I will update the naming in EVO-23.

File: app/containers/OrderPage/OrderDetailPage/index.tsx (hunk 2 above)
Comment (mrhodas107) [reply app/containers/OrderPage/OrderDetailPage/index.tsx]: The responsive behavior is not yet implemented.  That will take a little work.

**Pull request closed without merging**